streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -61,12 +62,13 @@
  *
  * This class do the translation between generic {@link Options} into {@link DBOptions} and {@link ColumnFamilyOptions}.
  */
-public class RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter extends Options {
+class RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter extends Options {
```

Comment:
I think this is not strictly needed since the constructor is already package-private.